kivy/resources.py

```diff
@@ -52,11 +52,11 @@ def resource_find(filename):
     if filename[:8] == 'atlas://':
         return filename
     if exists(filename):
-        return filename
+        return abspath(filename)
```

Comment:
There may be some subtle differences between `exists` and `abspath`, so that even if `exists(filename)`, `exists(abspath(filename))` may be False (although it probably is the same, but it never hurts to make sure). So it should probably be changed to `if exists(abspath(filename)): return abspath(filename)`. Similarly, for `output` below.